docs/manual/en/introduction/Creating-a-scene.html

```diff
@@ -43,10 +43,10 @@ <h2>Creating the scene</h2>
 		<p>To actually be able to display anything with three.js, we need three things: scene, camera and renderer, so that we can render the scene with camera.</p>
 
 		<code>
-		var scene = new THREE.Scene();
-		var camera = new THREE.PerspectiveCamera( 75, window.innerWidth / window.innerHeight, 0.1, 1000 );
+		let scene = new THREE.Scene();
```

Comment:
I would say `const` seems more appropriate here. Besides, there are other code sections in this document which should also be updated.

Reply:
Yep. `const` is the right one here.

Reply:
thanks! i made a new commit, to replace "let" with "const"

Reply:
Please also update the remaining `var` usage in this document.

Reply:
ok thanks, just did it